src/app/js/formats/streamgraph/Streamgraph.js

```diff
@@ -0,0 +1,410 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import * as d3 from 'd3';
+
+import { getCssToString, zoomFunction, distinctColors, transformDataIntoMapArray, randomId, getMinMaxValue } from './utils';
+import injectData from '../injectData';
+import exportableToPng from '../exportableToPng';
+
+const styles = {
```

Comment:
Please use the `aphrodite` library to manage your styles.
This way, the users will be able to custom it if needed.

Reply:
I use this styles in d3 method like this : `.attr("style", getCssToString(styles.tooltip, styles))`
I did this way because d3 needs the css as a string. I had a look to aphrodite and it doesn't seems to have a function to get the css as a string, it only want to inject css through the dom which is not good for this kind of use with d3. 

What do you think about it ? Did I miss something ?

(For now I can setup the css with aphrodite and use it with my method getCssToString, it works)

Reply:
Indeed, aphrodite generates class names.

I see that you're editing the `style` attribute to set inline CSS to it.
IMHO, you can do it the same, but with the `class` attribute in order to set class names to it.

For example:
```js
import { StyleSheet, css } from 'aphrodite/no-important';

const styles = StyleSheet.create({
    tooltip: { /* tooltip style */ },
});

// [...]
selector.attr('class', css(styles.tooltip));
```

Reply:
Absolutely yes, just didn't think to it.. thanks !